keyboards/keyhive/maypad/maypad.h

```diff
@@ -46,4 +46,16 @@
     { k40, KC_NO, k42, KC_NO }  \
 }
 
-#define LAYOUT LAYOUT_numpad_5x4
+#define LAYOUT_ortho_5x4( \
```

Comment:
I'd prefer if the identifiers were consistent between the two macros, which would make this:

```c
#define LAYOUT_ortho_5x4( \
    k00, k01, k02, k03, \
    k10, k11, k12, k13, \
    k20, k21, k22, k23, \
    k30, k31, k32, k33, \
    k40, k41, k42, k43  \
) { \
    { k00, k01, k02, k03 }, \
    { k10, k11, k12, k13 }, \
    { k20, k21, k22, k23 }, \
    { k30, k31, k32, k33 }, \
    { k40, k41, k42, k43 }  \
}
```

Reply:
Yeah, sorry I wasn't sure what numbers to fill in. I see now that it is k(row)(column), which makes WAY more sense.